src/DssSpell.sol

```diff
@@ -110,13 +110,33 @@ contract DssSpell {
     }
 
     modifier limited {
+        require(nextCastTime() == now, "Outside office hours");
```

Comment:
How do we feel about spaces in revert messages? I've always done them like `"Outside-office-hours"` because I had some idea about bot consumption and parsing, but I don't know if it holds any water.

Reply:
I doubt it matters much, but if we wanted to be consistent, we'd do something like `"DSSpell/outside-office-hours"`.

Reply:
Yeah I was following the convention from `schedule()`, I don't mind changing it, but we should change it across the board in that case.

Reply:
I'll just do it to all error messages with this convention: `"DSSpell/outside-office-hours"`